lib/libtlspool.c

```diff
@@ -275,9 +234,9 @@ static void registry_flush (pool_handle_t poolfd) {
 		if ((entry != NULL) && (entry->pfd != poolfd)) {
 			// Fill the cmd buffer with an error message
 			entry->buf->pio_cmd = PIOC_ERROR_V2;
-			entry->buf->pio_data.pioc_error.tlserrno = E_TLSPOOL_CLIENT_DISCONNECT;
+			entry->buf->pio_data.pioc_error.tlserrno = EPIPE;
 			strncpy (entry->buf->pio_data.pioc_error.message,
```

Comment:
Hier loop je achter op de invoering van com_err, de error is leesbaarder als E_TLSPOOL_CLIENT_DISCONNECT met als message "TLS Pool connection closed"